test/DebugInfo/value-generics-embedded.swift

```diff
@@ -1,16 +1,24 @@
-// RUN: %target-swift-frontend %s -target %target-cpu-apple-macos14 -emit-ir -g -enable-experimental-feature ValueGenerics -enable-experimental-feature Embedded -wmo -disable-availability-checking -o - | %FileCheck %s
+// RUN: %target-swift-frontend %s -target %target-cpu-apple-macos14 -emit-ir -gdwarf-types -enable-experimental-feature ValueGenerics -enable-experimental-feature Embedded -wmo -disable-availability-checking -o - | %FileCheck %s
```

Comment:
embedded + `-g` should emit it as `-gdwarf-types`, I think this should stay as `-g` to test this behavior.

Reply:
did you implement this logic in the driver only or also in the frontend? I think it may be necessary because this is the latter.

Reply:
I'm taking this back. It's not.